cms/datasets/utils.py

```diff
@@ -33,3 +41,94 @@ def format_datasets_as_document_list(datasets: StreamValue) -> list[dict[str, An
         dataset_documents.append(dataset_document)
 
     return dataset_documents
+
+
+def extract_edition_from_dataset_url(url: str) -> str | None:
+    """Extract the edition from a dataset URL.
+
+    Example URL: /datasets/wellbeing-quarterly/editions/september/versions/9
+    This would extract "september" as the edition.
+
+    Raises ValueError if the edition cannot be found in the URL.
```

Comment:
Needs updating, we now return None.

Reply:
Updated here: [18afc8c](https://github.com/ONSdigital/dis-wagtail/pull/389/commits/18afc8c470313d4cc3804d24c5beb1b54a7cc37c)